Sources/Brave/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -81,6 +84,21 @@ class TabLocationView: UIView {
       lockImageView.accessibilityLabel = Strings.tabToolbarLockImageAccessibilityLabel
     }
   }
+  
+  private func updateBravePlayerIcon() {
+    guard let url = url else {
+      bravePlayerButton.isHidden = true
+      return
+    }
+    
+    if ShieldPreferences.hasSeenAntiAdBlockWarning.value, let etldP1 = url.baseDomain, let components = URLComponents(url: url, resolvingAgainstBaseURL: false) {
```

Comment:
So is it correct user can only see Brave Player if user pressed "Adjust Shields For Me" button on youtube pop-over ?

Reply:
no, the user always sees brave player icon. The adjust shields is just informing the user they should disable shields for YouTube